### PR TITLE
Handle numbers in `styles`

### PR DESCRIPTION
**Pull request by acstll**
This would solve #18 

You can write

```js
h("div", { style: { opacity: 0 } })
```

and get

```html
<div style="opacity: 0;"></div>
```

### CODE REVIEW

File: src/patch.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function setElementProp(element, name, value, oldValue) {
   if (name === "key") {
   } else if (name === "style") {
     for (var name in merge(oldValue, (value = value || {}))) {
-      element.style[name] = value[name] || ""
+      element.style[name] = value[name] != null ? value[name] : ""
```
Comment (JorgeBucaran) [inline src/patch.js]: @acstll Could we get away with?

```js
element.style[name]+=""
```

Comment (acstll) [reply src/patch.js]: With `element.style[name]+=""` tests fail.

`!= null` checks against `null` and `undefined`, so `0` is valid. Anything else will be already coerced into a string by the browser.

If I'm correct, this change will keep things intact and only allow `0` to be valid and not being swallowed as falsy. Everything else stays the same.

Comment (JorgeBucaran) [reply src/patch.js]: Sorry, I wasn't thinking well! 😅

**Comment by JorgeBucaran** [comment]
Thanks @acstll! 👍

**Comment by acstll** [comment]
You're welcome!

**Pull request merged**